test/functional/ex_cmds/dict_notifications_spec.lua

```diff
@@ -338,4 +339,25 @@ describe('dictionary change notifications', function()
       eq({'notification', '2', {'foo', {old = 'baz', new = 'bar'}}}, next_msg())
     end)
   end)
+
+  describe('Event For changedtick of b:', function()
+    it('with keypress', function()
+        source([[
+          function! OnTickChanged(dict, key, value)
+              call rpcnotify(g:channel, 'SendChangeTick', a:key, a:value)
+          endfunction
+          call dictwatcheradd(b:, "changedtick", "OnTickChanged")
+        ]])
+
+        insert("t");
+
+        local res = next_msg()
+        assert.is_true(res[2] == "SendChangeTick")
```

Comment:
can use `helpers.eq()` to compare the values, which gives more informative failures.

Also better to compare the full `next_msg()` result, not just `[2]`.

I made these changes in the merge-commit.

Reply:
ok, understood